daemon/mgr/container_diskquota.go

```diff
@@ -0,0 +1,117 @@
+package mgr
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/alibaba/pouch/pkg/quota"
+	"github.com/docker/docker/pkg/reexec"
+	"github.com/sirupsen/logrus"
+)
+
+func init() {
+	reexec.Register("set-diskquota", processSetQuotaReexec)
+}
+
+func processSetQuotaReexec() {
+	var (
+		err      error
+		upperDir string
+	)
+
+	// Return a failure to the calling process via ExitCode
+	defer func() {
+		if err != nil {
+			logrus.Fatalf("%v", err)
+		}
+	}()
+
+	if len(os.Args) != 3 {
+		err = fmt.Errorf("command args is invalid: %v", os.Args)
```

Comment:
I suggest that when we report the invalidness to user or in log, we could just walk one step further to tell others why it is invalid, or the valid type. It will bring much convenience to our dear users.